slowmovie.py

```diff
@@ -210,6 +211,17 @@ def estimate_runtime(delay, increment, frames, output="guess"):
 consoleHandler.setFormatter(logging.Formatter("%(message)s"))
 logger.addHandler(consoleHandler)
 
+# check the epd driver - do this first since we can't do much if it fails
+validEpds = displayfactory.list_supported_displays()
+
+if(args.epd not in validEpds):
+    # can't find the driver
+    logging.error(f"{args.epd} is not a valid EPD name, valid names are:")
+    logging.error("\n".join(map(str, validEpds)))
+
+    # can't get past this
+    exit(1)
```

Comment:
Should this be `sys.exit()` now? I think @missionfloyd standardized the main code to use that a bit ago.

Reply:
Yes, `exit()` is for the command line interpreter.